pkg/kubelet/kubelet.go

```diff
@@ -2678,12 +2678,20 @@ func getPodReadyCondition(spec *api.PodSpec, containerStatuses []api.ContainerSt
 		unreadyMessages = append(unreadyMessages, fmt.Sprintf("containers with unknown status: %s", unknownContainers))
 	}
 	if len(unreadyContainers) > 0 {
-		unreadyMessages = append(unreadyMessages, fmt.Sprintf("containers with unready status: %s", unreadyContainers))
+		if podPhase == api.PodSucceeded {
+			unreadyMessages = append(unreadyMessages, fmt.Sprintf("containers terminated success: %s", unreadyContainers))
```

Comment:
Maybe an empty message is fine in this case?  No point in listing the unready container names since they aren't expected to be ready.